cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/cats/sql/controllers/CatsSqlAdminController.kt

```diff
@@ -102,7 +93,43 @@ class CatsSqlAdminController(
     return CleanTablesResult(tableCount = tablesDropped.size, tables = tablesDropped)
   }
 
-  private fun validateParams(currentNamespace: String?, targetNamespace: String) {
+  @PutMapping(path = ["drop_version/{version}"])
+  fun dropTablesByVersion(@PathVariable("version") dropVersion: String): CleanTablesResult {
```

Comment:
You can make this the enum type as a parameter and then Spring will do the validation work for you. I'd feel more comfortable using the enum as the path variable type so we don't have any unwelcome SQL injection regressions in the future.

Reply:
Sounds good, done. Thanks!